projects/extension/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@substrate/extension",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
This should be `0.1.1`
We didn't do any breaking change to the API of the extension